warplib/src/warp_memory/bytes.cairo

```diff
@@ -1,6 +1,62 @@
+use integer::u128_to_felt252;
+use integer::u256_from_felt252;
+use warplib::integer::u256_from_felts;
+use warplib::integer::u256_into_felt_unsafe;
 use warplib::warp_memory::WarpMemory;
 use warplib::warp_memory::WarpMemoryTrait;
 
 trait WarpMemoryBytesTrait{
     fn bytes_to_fixed_bytes32(ref self: WarpMemory, location: felt252) -> u256;
+    fn wm_bytes_to_fixed_bytesN(self: WarpMemory, bytesLoc: felt252, width: felt252) -> felt252 ;
 }
+
+impl WarpMemoryBytesImpl of WarpMemoryBytesTrait{
+    fn bytes_to_fixed_bytes32(self: WarpMemory, bytesLoc: felt252) -> u256 {
+        // TODO Is this self call to read_felt enough, don't we need to import from WarpMemoryReadTrait??
+        let data_len = self.read_u256(bytesLoc);
```

Comment:
length from bytes is now stored as a felt instead of a u256. You should read a single felt from here